var/spack/repos/builtin/packages/fermisciencetools/package.py

```diff
@@ -0,0 +1,42 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
+from spack import *
+from distutils.dir_util import copy_tree
+
+
+class Fermisciencetools(Package):
+    """The Fermi Science Tools consists of the basic tools necessary to
+    analyze Fermi data. 
+    
+    This is the binary version for Linux x86_64 with libc-2.17."""
```

Comment:
It looks like source distributions of this software are also available. You should build this from source to get better performance.

Reply:
I tried the source version, but it seems like there might be some logical errors in the configure codes. It finds the dependency in the first step, and then it cannot find it in the next step. I cannot install it with the source version, so I used the binary one for now.

Reply:
I see. Can you add a comment mentioning this? Hopefully someone else can figure it out someday.

All you need to do is fix the flake8 warning and I can merge.

Reply:
Added comments and fixed the flake8.